examples/radiation.ipynb

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Can you rename the Notebook with a capital letter "Radiation"? The other notebooks have capital letters.